Cargo.toml

```diff
@@ -22,12 +22,19 @@ features = ["serde-serialize"]
 test = false
 
 [features]
-default = ["std"]
+default = ["std", "msrv"]
 enable-interning = ["std"]
 serde-serialize = ["serde", "serde_json", "std"]
 spans = []
 std = ["wasm-bindgen-macro/std", "once_cell/std"]
 
+# Opt-in for Rust language features that require a higher MSRV.
+#
+# The current rustc version is detected at compile-time, so enabling this
+# feature for older compilers will NOT result in a compilation error. Instead,
+# any unsupported language feature will not be used.
+msrv = ["rustversion"]
```

Comment:
When https://github.com/rustwasm/wasm-bindgen/pull/4345 is merged, add this crate feature to these crates as well and then add it to the test matrix.

Reply:
Done.